### ISSUE TITLE
WebEngine: Tabs open in background even without background-tabs

### ISSUE THREAD

**Issue by The-Compiler** [The-Compiler/qutebrowser#1956]
When doing `QUTE_QTBUG54419_PATCHED=1 ./.venv/bin/qutebrowser --debug --temp-basedir --backend webengine` middle-clicked links are opened in background.

It seems like they're handled by `createWindow` despite us not wanting that:

```
09:39:42 DEBUG    mouse      mouse:_mousepress_opentarget:218 Ctrl/Middle click, setting target: ClickTarget.tab
[...]
09:39:42 DEBUG    webview    webview:createWindow:68 createWindow with type WebBrowserBackgroundTab
09:39:42 DEBUG    webview    tabbedbrowser:tabopen:372 Creating new tab with URL None
[...]
09:39:42 DEBUG    webview    browsertab:_set_load_status:590 load status for <qutebrowser.browser.webengine.webenginetab.WebEngineTab tab_id=1 url=''>: LoadStatus.loading
[...]
09:39:42 DEBUG    webview    webview:acceptNavigationRequest:153 navigation request: url http://qutebrowser.org/CHANGELOG.html, type NavigationTypeLinkClicked, target ClickTarget.normal, is_main_frame True
09:39:42 DEBUG    webview    browsertab:_on_link_clicked:619 link clicked: url http://qutebrowser.org/CHANGELOG.html, override target None, open_target ClickTarget.normal
```

**Comment by lluixhi** [comment]
Odd. I have background-tabs set to true, tabs are not opening in the background on middle click.

Would this be for the same reason?

On QtWebEngine 5.6.2

**Comment by The-Compiler** [comment]
Should be fixed by 5de07246be9e0c34ebdad8a6da297b50ee6b2826.
